tests/ui/test_login.py

```diff
@@ -1,23 +1,24 @@
+import os
 import pytest
 
 from pages.desktop.home import Home
 
 
-@pytest.mark.skip(
-    reason='https://github.com/mozilla/addons-server/issues/2462')
-def test_login(base_url, selenium, user):
+@pytest.mark.skipif('localhost' in os.environ.get('PYTEST_BASE_URL'),
+                    reason='Live Server login currently not functioning')
```

Comment:
The `PYTEST_BASE_URL` is not the only place the base URL can be set, so we shouldn't use that to check the value. Also, 'localhost' does not mean the live server is in use - an instance may be running independently on the local machine.